msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -48,7 +48,7 @@ An API translation layer (mapping v.1 to v.2 TeamsJS API calls) is provided to e
 
 #### Teams-only apps
 
-Even if you intend your app to only run in Teams (and not Microsoft 365 app and Outlook), best practice is to start referencing the latest TeamsJS (*v.2.0* or later) as soon as convenient, in order to benefit from the latest improvements, new features, and support (even for Teams-only apps). TeamsJS v.1.12 will continue to be supported, but no new features or improvements will be added.
+Even if you intend your app to only run in Teams (and not Microsoft 365 app and Outlook), best practice is to start referencing the latest TeamsJS (*v.2.0* or later) as soon as convenient, in order to benefit from the latest improvements, new features, and support (even for Teams-only apps). We recommend you to use TeamsJS v.1.13.1. TeamsJS v.1.12 continues to be supported, but no new features or improvements are added.
```

Comment:
> We recommend you to use TeamsJS v.1.13.1. TeamsJS v.1.12 continues to be supported, but no new features or improvements are added.

This seems confusing. We are recommending v1.13.1 so same should be mentioned for continues support rather than old version. 

**Updated version:** 
We recommend you to use TeamsJS v.1.13.1. This version continues to be supported, but no new features or improvements are added.

Reply:
Updated as suggested